Invalid memory address or nil pointer dereference when running in k8s

I'm getting this error when running a test from a k8s cluster, but not when running locally through CLI non-container. The script only uses 1 VU.

k6v 0.25.1

Looks similar to this issue fixed earlier this year https://github.com/loadimpact/k6/issues/825

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: Panic at 35: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: Panic at 35: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: Panic at 10: Panic at 35: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: Panic at 10: Panic at 35: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x9f7145]

goroutine 77 [running]:
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1.1(0xc003541d30)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1407 +0x98
panic(0xd45020, 0xc000d4a450)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).try.func1(0xc004d960d0, 0x0, 0xc003541c00, 0x0, 0x0, 0x0, 0xc003541c58)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:364 +0x496
panic(0xd45020, 0xc000d4a430)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1.1(0xc003541278)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1407 +0x98
panic(0xd45020, 0xc000d4a430)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).try.func1(0xc004d960d0, 0x3, 0xc003541148, 0x12, 0x0, 0x0, 0xc0035411a0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:364 +0x496
panic(0xd59960, 0x192dc00)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/loadimpact/k6/lib/netext/httpext.(*Tracer).GotConn(0xc004beda00, 0x0, 0x0, 0xc007000101, 0xf1c04087b)
	/go/src/github.com/loadimpact/k6/lib/netext/httpext/tracer.go:230 +0x45
net/http.(*Transport).getConn(0xc004e30000, 0xc004551d10, 0x0, 0xc00272dbc0, 0x5, 0xc005e46300, 0x10, 0x0, 0x0, 0x0, ...)
	/usr/local/go/src/net/http/transport.go:1210 +0x394
net/http.(*Transport).roundTrip(0xc004e30000, 0xc00177ca00, 0xe24d80, 0xc0041f8f01, 0xc004bedb00)
	/usr/local/go/src/net/http/transport.go:522 +0x603
net/http.(*Transport).RoundTrip(0xc004e30000, 0xc00177ca00, 0xc004beda80, 0x10aa300, 0xc004436630)
	/usr/local/go/src/net/http/roundtrip.go:17 +0x35
github.com/loadimpact/k6/lib/netext/httpext.(*transport).RoundTrip(0xc0041f85a0, 0xc00177c800, 0xc0041f85a0, 0xbf5678e4dc20aec2, 0x1e53b4209d)
	/go/src/github.com/loadimpact/k6/lib/netext/httpext/transport.go:191 +0x190
net/http.send(0xc00177c700, 0x1097500, 0xc0041f85a0, 0xbf5678e4dc20aec2, 0x1e53b4209d, 0x19453c0, 0xc0041f8800, 0xc0035401e0, 0x4, 0x10)
	/usr/local/go/src/net/http/client.go:250 +0x43a
net/http.(*Client).send(0xc00443df80, 0xc00177c700, 0xbf5678e4dc20aec2, 0x1e53b4209d, 0x19453c0, 0xc0041f8800, 0x2, 0x4, 0x0)
	/usr/local/go/src/net/http/client.go:174 +0xfa
net/http.(*Client).do(0xc00443df80, 0xc00177c700, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/http/client.go:641 +0x3ce
net/http.(*Client).Do(...)
	/usr/local/go/src/net/http/client.go:509
github.com/loadimpact/k6/lib/netext/httpext.MakeRequest(0x10aa300, 0xc004a2faa0, 0xc004bed680, 0xe869c1, 0x3, 0xc004bed600)
	/go/src/github.com/loadimpact/k6/lib/netext/httpext/request.go:288 +0x816
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).Request(0xc0000fa000, 0x10aa300, 0xc004a2faa0, 0xe869c1, 0x3, 0x10b70c0, 0xc000de8530, 0xc0041f84c0, 0x2, 0x2, ...)
	/go/src/github.com/loadimpact/k6/js/modules/k6/http/request.go:111 +0x17d
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).Get(0xc0000fa000, 0x10aa300, 0xc004a2faa0, 0x10b70c0, 0xc000de8530, 0xc000de8550, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/loadimpact/k6/js/modules/k6/http/request.go:53 +0x15b
reflect.Value.call(0xe221c0, 0xc0000fa000, 0x1213, 0xe8c2e5, 0x9, 0xc0046e7cc0, 0x3, 0x3, 0xb, 0xdf1340, ...)
	/usr/local/go/src/reflect/value.go:460 +0x5f6
reflect.Value.CallSlice(0xe221c0, 0xc0000fa000, 0x1213, 0xc0046e7cc0, 0x3, 0x3, 0x194, 0xc003540c98, 0x8c29d3)
	/usr/local/go/src/reflect/value.go:334 +0xb4
github.com/loadimpact/k6/js/common.Bind.func1(0x10b6ee0, 0xc00389fd60, 0xc006356220, 0x2, 0xbc, 0x409b1b, 0x10)
	/go/src/github.com/loadimpact/k6/js/common/bridge.go:239 +0x780
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm)._nativeCall(0xc004d960d0, 0xc00291d1e0, 0x2)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:1826 +0x291
github.com/loadimpact/k6/vendor/github.com/dop251/goja.call.exec(0xffffffff00000002, 0xc004d960d0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:1810 +0x4a2
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).run(0xc004d960d0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:288 +0x51
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*funcObject).Call(0xc004885680, 0x10b7520, 0x19643a0, 0x0, 0x0, 0x0, 0x10b7520, 0x19643a0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/func.go:130 +0x2cc
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1.2()
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1412 +0x96
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).try(0xc004d960d0, 0xc0035411f0, 0x0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:370 +0x147
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1(0x10b7520, 0x19643a0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1411 +0x13a
github.com/loadimpact/k6/js/modules/k6.(*K6).Group(0x19645f8, 0x10aa300, 0xc004a2faa0, 0xc003ec854b, 0x9, 0xc004201d40, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/loadimpact/k6/js/modules/k6/k6.go:90 +0x1c9
reflect.Value.call(0xda0d60, 0x19645f8, 0xa13, 0xe86e70, 0x4, 0xc0046e6f50, 0x3, 0x3, 0xce11e0, 0xc00000e050, ...)
	/usr/local/go/src/reflect/value.go:460 +0x5f6
reflect.Value.Call(0xda0d60, 0x19645f8, 0xa13, 0xc0046e6f50, 0x3, 0x3, 0x0, 0xe48f40, 0xc004201c60)
	/usr/local/go/src/reflect/value.go:321 +0xb4
github.com/loadimpact/k6/js/common.Bind.func1(0x10b7520, 0x19643a0, 0xc006356100, 0x2, 0xce, 0xc004838fc0, 0xc003541968)
	/go/src/github.com/loadimpact/k6/js/common/bridge.go:241 +0x9bc
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm)._nativeCall(0xc004d960d0, 0xc00291c8f0, 0x2)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:1826 +0x291
github.com/loadimpact/k6/vendor/github.com/dop251/goja.call.exec(0x2, 0xc004d960d0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:1810 +0x4a2
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).run(0xc004d960d0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:288 +0x51
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*funcObject).Call(0xc000c2b800, 0x10b7520, 0x19643a0, 0xc00320c2c0, 0x1, 0x1, 0x10b7520, 0x19643a0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/func.go:130 +0x2cc
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1.2()
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1412 +0x96
github.com/loadimpact/k6/vendor/github.com/dop251/goja.(*vm).try(0xc004d960d0, 0xc003541ca8, 0x0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/vm.go:370 +0x147
github.com/loadimpact/k6/vendor/github.com/dop251/goja.AssertFunction.func1(0x10b7520, 0x19643a0, 0xc00320c2c0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/loadimpact/k6/vendor/github.com/dop251/goja/runtime.go:1411 +0x13a
github.com/loadimpact/k6/js.(*VU).runFn(0xc0011c73b0, 0x10aa240, 0xc0019606c0, 0xc003060b40, 0xc000f03e60, 0xc00320c2c0, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/loadimpact/k6/js/runner.go:444 +0x392
github.com/loadimpact/k6/js.(*VU).RunOnce(0xc0011c73b0, 0x10aa240, 0xc0019606c0, 0x0, 0x0)
	/go/src/github.com/loadimpact/k6/js/runner.go:404 +0x202
github.com/loadimpact/k6/core/local.(*vuHandle).run(0xc004d69000, 0xc000126000, 0xc004b6c4e0, 0xc0000aa5a0)
	/go/src/github.com/loadimpact/k6/core/local/local.go:71 +0x152
github.com/loadimpact/k6/core/local.(*Executor).scale.func1(0xc004d69000, 0xc004d8f7a0, 0xc004b6c4e0, 0xc0000aa5a0)
	/go/src/github.com/loadimpact/k6/core/local/local.go:361 +0x4d
created by github.com/loadimpact/k6/core/local.(*Executor).scale
	/go/src/github.com/loadimpact/k6/core/local/local.go:360 +0x328
npm ERR! code ELIFECYCLE
npm ERR! errno 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine as builder
+FROM golang:1.13-alpine as builder
 WORKDIR $GOPATH/src/github.com/loadimpact/k6
 ADD . .
 RUN apk --no-cache add --virtual .build-deps git make build-base && \

